apps/api/src/server/applications/examination-timetable-items/examination-timetable-items.service.js

```diff
@@ -31,34 +31,39 @@ function extractDescriptionAndLineItems(examinationTimetableItem) {
 
 	const parsedDescription = JSON.parse(examinationTimetableItem.description);
 
-	// The event line items are the bullet points that are added to the description. They are different from ExaminationTimetableItems.
-	const eventLineItems = parsedDescription.bulletPoints.map((/** @type {string} */ bulletPoint) => {
-		return {
-			eventLineItemDescription: bulletPoint
-		};
-	});
-
 	return {
 		description: parsedDescription.preText,
-		eventLineItems
+		eventLineItems: parsedDescription.bulletPoints
 	};
 }
 
 /**
  * Returns the payload containing all the examination timetable items.
  *
  * @param {number} examinationTimetableId
- * @returns { Promise<NSIPExamTimetableItem[]> }
+ * @returns { Promise<NSIPExamTimetable | null> }
  */
 async function buildExamTimetableItemsPayload(examinationTimetableId) {
-	const examinationTimetableItems =
-		await examinationTimetableItemsRepository.getByExaminationTimetableId(examinationTimetableId);
+	const examinationTimetable = await examinationTimetableRepository.getWithItems(
+		examinationTimetableId
+	);
+	if (!examinationTimetable) {
+		return null;
+	}
 
-	if (!examinationTimetableItems) {
-		return [];
+	const caseObject = await caseRepository.getById(examinationTimetable.caseId, {});
```

Comment:
We can probably add the case { reference } as an include in the examinationTimetableRepository.getWithItems call